CHANGELOG.md

```diff
@@ -31,8 +31,15 @@ v2.0.2 is a feature release with the following features, fixes and enhancements:
  - Improved Serialization and Deserialization Examples.
  - Documentation Improvements.
 
+## Upgrade considerations
+
+OpenSSL 3.0.x upgrade in librdkafka requires a major version bump, as some
+ legacy ciphers need to be explicitly configured to continue working,
+ but it is highly recommended NOT to use them. The rest of the API remains
+ backward compatible. See librdkafka release notes below.
```

Comment:
I think its better to remove "See librdkafka release notes below." as we are saying the same in the next line.